security/providers/http-auth/src/main/java/io/helidon/security/providers/httpauth/HttpDigestAuthProvider.java

```diff
@@ -342,6 +351,11 @@ public HttpDigestAuthProvider build() {
          * @param subjectType type of principal
          * @return updated builder instance
          */
+        @ConfiguredOption(key = "principal-type",
+                          value = "USER",
+                          description = "Type of authenticated entity, can be used in combination with other authentication "
```

Comment:
Please either use javadocs, or update javadocs to new documentation. The description is not required here and duplicates information.